Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,303 @@
+        @Published var inputBoxIsPresented = false
+        
+        /// The input obtained from the user for the buffer radius of a point.
+        @Published var radiusInput: String = ""
+        
+        /// The status of the sample.
+        @Published var status = Status.addPoints
+        
+        /// A map centered on Texas with image layers.
+        let map: Map
+        
+        /// The graphics overlays used in this sample.
+        var graphicsOverlays: [GraphicsOverlay] = []
+        
+        /// An array of the tapped points and their radii.
+        var bufferPoints: [(point: Point, radius: Double)] = []
+        
+        /// The graphics overlay for the boundary around the valid area of use.
+        private var boundaryGraphicsOverlay: GraphicsOverlay
+        
+        /// The graphics overlay for the points' buffers.
+        private var bufferGraphicsOverlay: GraphicsOverlay
+        
+        /// The graphics overlay for the points of the tapped locations.
+        private var tapPointsGraphicsOverlay: GraphicsOverlay
+        
+        /// A polygon that represents the valid area of use for the spatial reference.
+        private let boundaryPolygon: ArcGIS.Polygon
+        
+        init() {
+            /// The spatial reference for the sample.
+            let statePlaneNorthCentralTexas = SpatialReference(wkid: WKID(32038)!)!
+            
+            // Create boundary polygon.
+            boundaryPolygon = {
+                let boundaryPoints = [
+                    Point(latitude: 31.720, longitude: -103.070),
+                    Point(latitude: 34.580, longitude: -103.070),
+                    Point(latitude: 34.580, longitude: -94.000),
+                    Point(latitude: 31.720, longitude: -94.000)
+                ]
+                let polygon = GeometryEngine.project(Polygon(points: boundaryPoints), into: statePlaneNorthCentralTexas)!
+                return polygon
+            }()
+            
+            // Create map.
+            map = CreateBuffersAroundPointsView.Model.makeMap(
+                spatialReference: statePlaneNorthCentralTexas,
+                viewpointGeometry: boundaryPolygon
+            )
+            
+            // Create graphics overlays.
+            boundaryGraphicsOverlay = CreateBuffersAroundPointsView.Model.makeBoundaryGraphicsOverlay(
+                boundaryGeometry: boundaryPolygon
+            )
+            bufferGraphicsOverlay = CreateBuffersAroundPointsView.Model.makeBufferGraphicsOverlay()
+            tapPointsGraphicsOverlay = CreateBuffersAroundPointsView.Model.makeTappedPointsGraphicsOverlay()
+            graphicsOverlays.append(
+                contentsOf: [boundaryGraphicsOverlay, bufferGraphicsOverlay, tapPointsGraphicsOverlay]
+            )
+        }
+        
+        /// Create a map with image layers from a spatial reference.
+        /// - Parameters:
+        ///   - spatialReference: The 'SpatialReference' the 'Map' is derived from.
```

Comment:
```suggestion
        ///   - spatialReference: The `SpatialReference` the `Map` is derived from.
```
|Before|After|
|:-:|:-:|
|<img width="562" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/663a52d3-775f-4cac-8a97-9c2b4dd57e8e">|<img width="562" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/34db180e-c43a-4566-b811-ff8d65b843be">|

Use backtick or plain English for markup docstring. Please fix other places as well.